### PR TITLE
Update ja.json

### PR DESCRIPTION
**Pull request by tyoripoko**

### CODE REVIEW

File: translation/ui/ja.json [hunk 1]
```diff
@@ -1,2993 +1,6129 @@
...
+    "location_sig_gorge_skull": "ク＝ガスの頭蓋骨",
+    "display_name_bw_gate_0006": "成就の門",
     "mission_skittergate_kill_rasknitt": "ラスクニットを倒せ",
-    "dr_1h_throwing_axes_skin_02_magic_01_description": "魔力の霊魂が注がれた武器。",
+    "input_description_play": "プレイ",
+    "es_1h_flail_skin_01_name": "裁定者のフレイル",
+    "display_name_bountyhunter_hat_0007": "彗星の刻印",
+    "input_description_connect": "接続",
     "bw_1h_crowbill_skin_01_magic_01_description": "〈魔力の流れ〉の霊魂が注がれた武器。",
+
```
Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: Syntax error

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: Syntax error, check this char `�`

Comment (Iskied) [inline translation/ui/ja.json]: Syntax error

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`.

Comment (Iskied) [inline translation/ui/ja.json]: You should change `�` this character to `\n`. Don't change %s to %, this will be crashing the game

Comment (Iskied) [inline translation/ui/ja.json]: Do you mean `\"`?

File: translation/ui/ja.json [hunk 2]
```diff
@@ -1,2993 +1,6129 @@
...
+    "wh_repeating_pistol_skin_03_description": "リードショットで繰り返し叩くような異端者からでんぷんを取り除くものはありません。この拳銃の各バレルは、シグマーライトの美徳の1つにちなんで名付けられています。 「忍耐」はその中にはありません。",
+    "we_longbow_skin_06_runed_02_description": "ハンターを回避することはできません。アスライの復讐も。あなたが逃げるところならどこでも矢はあなたを見つけ、あなたの体は空腹の土に与えられるでしょう。",
+    "hero_view_perk_title": "特性",
+    "description_properties_stamina": "最大スタミナを増加させる",
+    "achv_holly_magnus_rune_name": "マグナスの角笛ルーン",
+    "description_slayer_hat_0012": "より短いより厚いスラーのためのより短いより厚いナール。",
+    "display_name_default_trinket": "宝石職人の小物",
+    "location_bastion_auditorium": "呪われた聖域",
+    "achv_scorpion_crater_dark_tongue_2_desc": "ビーストマンの隠れ家を見つけて侵入する",
+    "loading_screen_tip_09": "[$KEY;Player__wield_switch_1 :]キーを押すと、射撃武器に切り替わる。",
```
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

File: translation/ui/ja.json [hunk 3]
```diff
@@ -1,2993 +1,6129 @@
...
+    "bardin_ranger_passive_spawn_potions_or_bombs": "廃品回収",
+    "we_life_staff_skin_02_runed_02_name": "贖い主のサリオシ",
+    "es_sword_shield_breton_skin_05_magic_01_description": "剣を打つたびに歴史を刻み、盾の下に真実を隠したパラヴォンのガウェインに属する伝説の腕。",
+    "loading_screen_tip_27": "[$KEY;Player__action_one :]キーを長押しすると、近接攻撃をチャージできる。",
```
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

File: translation/ui/ja.json [hunk 4]
```diff
@@ -1,2993 +1,6129 @@
...
-    "mission_crater_follow_beastmen_path": "獣道をたどれ",
-    "mission_bogenhafen_city_pushforward2": "水門をくぐり抜けろ！",
-    "mission_fort_use_cannon_01": "発射台で大砲を撃て",
-    "mission_mines_find_explosives": "黒火薬を見つけろ",
-    "mission_farmlands_free_prisoners_04": "のこぎり水車小屋のすべての囚人を解放しろ",
-    "mission_mines_collect_explosives": "黒火薬を集めろ",
-    "mission_bogenhafen_slum_survive_hideout": "生き延びろ！",
+    "display_name_ww_hood_0004": "ドゥラキラの角",
+    "achv_complete_all_helmgart_levels_champion_wh_bountyhunter_name": "疫病裁判Ⅲ:賞金稼ぎ",
+    "loading_screen_tip_84": "レバースイッチやマップ内の様々な仕掛けは、[$KEY;Player__interact :]キーによって動かすことが出来る。",
```
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

File: translation/ui/ja.json [hunk 5]
```diff
@@ -1,2993 +1,6129 @@
...
+    "display_name_default_dwarf_ranger_dr_handgun": "デブスルンド",
+    "career_passive_name_wh_1c": "鋼鉄の信念",
+    "we_longbow_skin_03_name": "捕食者のナスティラス",
+    "display_name_zealot_hat_0000": "鞭打苦行者の焼印",
+    "frame_geheimnisnacht_02_name": "ファンガスティックな不屈の精神",
+    "victor_zealot_passive_healing_received": "聖なる不屈の精神",
+    "location_elven_maze": "グリスバウムの沼",
+    "achv_achievement_kerillian_level_1_name": "緑葉の衣",
+    "achv_complete_all_helmgart_levels_legend_we_waywatcher_desc": "ウェイストーカーにて、難易度:伝説ですべてのヘルムガートミッションを達成する",
+    "loading_screen_tip_45": "[$KEY;Player__action_one :]で通常攻撃、[$KEY;Player__action_one :]長押しでチャージ攻撃となる。",
```
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

File: translation/ui/ja.json [hunk 6]
```diff
@@ -1,2993 +1,6129 @@
...
+    "es_longbow_skin_05_runed_02_description": "弓はシグマーの時代からさかのぼると言われていますが、それは真実よりも寓話のように思えます。いずれにせよ、それは挑戦的な引き分けをマスターできる誰にとっても強力な戦いの道具であり続けます。",
+    "lb_game_type_twitch": "Twitch",
+    "display_name_ring_06": "従者の輪",
+    "sienna_adept_attack_speed_on_enemies_hit": "焼身自殺依存",
+    "location_stronghold_warlord_nest": "背骨潰しのスカーリックの隠れ家",
+    "description_scholar_hat_0011": "決闘者の敗北を記念して金の大学からの贈り物。その存在下でかすかな笑い声が聞こえます。",
+    "achv_complete_skittergate_hard_desc": "難易度:古参兵でスキッターゲートを達成する",
+    "description_es_helmet_0003": "このフルバイザードヘルメットは、マグナスタワーを中心とする武ベルスライクの軍管区の境内にある武器庫で発見されました。町の防御は、ハルバードの12個分遣隊と、監視塔に沿って設置された砲兵用の8個の石弓メンと3個のエンジニアユニットで構成されていました。 ����ディフェンダーはスケイヴンスニークアタックで大きな損失を被りましたが、少なくとも一部はドワーフェン地区と抵抗のアンを形成した丘に逃げることができました。彼らの指揮官キャプテンアーウィンブルーチャーは行方不明であると報告されています。",
+    "career_passive_name_we_1": "殺人技術",
+    "loading_screen_tip_01": "後方または横に移動しながら[$KEY;Player__jump_1 :]を押すことで、敵の攻撃を回避したり、間合いを開くことができる。",
```
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

File: translation/ui/ja.json (hunk 1 above)
Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

Comment (Iskied) [inline translation/ui/ja.json]: Don't change macro string.

**Review by Iskied — CHANGES_REQUESTED** [review]
Check review

File: translation/ui/ja.json [hunk 7]
```diff
@@ -1,2993 +1,6129 @@
...
+    "dr_deus_01_skin_02_description": "チェーンだけがその使用者からの命令なしに発射を防ぐことができるような怒りのトロルハンマー魚雷発射管（またはフレンドリーファイアの後にバーディンが主張する）。",
+    "achv_cog_all_kill_barrage_desc": "モンスター、特殊、精鋭兵を、1回のクランク銃射撃で倒す。",
+    "loading_screen_tip_47": "どのプレイヤーも、プレイインターフェースを通じて新しいミッションへの投票を開始できる。",
+    "wh_1h_falchion": "ファルシオン",
+    "crafting_material_dust_2": "天空の塵",
+    "display_name_deus_knockdown_damage_immunity_aura": "モールの守護",
+    "achv_gecko_scraps_skittergate_2_name": "解き放たれし怒りの影",
+    "bardin_ranger_passive_spawn_potions_or_bombs_desc": "「残りし者」の効果にて、20%の確率で弾薬袋の代わりにポーションや爆弾をドロップするようになる。",
+    "achv_skaven_poison_wind_globardier_3_name": "上物を一服",
+    "description_dr_helmet_0011": "「ボガ」とは、バーディンがよく用いるカザリッド語の表現で、「ろうそくを消してトンネルを突然真っ暗にする」を意味する。 ����この鉱夫のヘルメットの名は、弾力性に優れたドワーフ特性のろうそくのしなやかさを指す言葉と同じく「ネバーボガ」と訳される。 ����これまで彼らはその名に忠実だったが、バーディンのろうそくが底を尽いてからというもの、高級なドワーフろうそくを買いだめするまでの間、彼がこのヘルメットを被るかどうかは疑わしい。なぜなら、ウムガク・マンリング製の品は、単に信頼できないからだ。",
```
[the remark this answers is not in the document]
Comment (tyoripoko) [reply translation/ui/ja.json]: ok

**Review by Iskied — COMMENTED** [review]
Closed

**Pull request closed without merging**